src/pages/pt-BR/editor-setup.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: ~/layouts/MainLayout.astro
+setup: |
+  import Badge from '~/components/Badge.astro';
+title: Configuração do Editor
+description: Configure seu editor para desenvolver com Astro.
+i18nReady: true
+---
+
+Customize seu editor de código para melhorar a sua experiência de desenvolvimento com Astro e desfrute de novas funcionalidades.
+
+## VS Code
+
+[VS Code](https://code.visualstudio.com) é um popular editor de código para desenvolvedores web, feito pela Microsoft. O motor do VS Code também viabiliza populares editores de código no navegador como o [GitHub Codespaces](https://github.com/features/codespaces) e o [Gitpod](https://gitpod.io).
```

Comment:
I think I would change the adjective position here:
```suggestion
[VS Code](https://code.visualstudio.com) é um popular editor de código para desenvolvedores web, feito pela Microsoft. O motor do VS Code também viabiliza editores de código populares no navegador como o [GitHub Codespaces](https://github.com/features/codespaces) e o [Gitpod](https://gitpod.io).
```

Reply:
Thanks!